backend_new/prisma/seed-helpers/listing-factory.ts

```diff
@@ -0,0 +1,388 @@
+      schools: `listingNeighborhoodAmenities: ${i} schools: ${i}`,
+      publicTransportation: `listingNeighborhoodAmenities: ${i} publicTransportation: ${i}`,
+    },
+  },
+  verifiedAt: new Date(),
+  homeType: HomeTypeEnum.apartment,
+  region: RegionEnum.Greater_Downtown,
+  // end detroit specific
+
+  applicationMethods: {
+    create: {
+      type: ApplicationMethodsTypeEnum.Internal,
+      label: `applicationMethods: ${i} label: ${i}`,
+      externalReference: `applicationMethods: ${i} externalReference: ${i}`,
+      acceptsPostmarkedApplications: true,
+      phoneNumber: `applicationMethods: ${i} phoneNumber: ${i}`,
+    },
+  },
+  listingEvents: {
+    create: {
+      type: ListingEventsTypeEnum.publicLottery,
+      startDate: new Date(),
+      startTime: new Date(),
+      endTime: new Date(),
+      url: `listingEvents: ${i} url: ${i}`,
+      note: `listingEvents: ${i} note: ${i}`,
+      label: `listingEvents: ${i} label: ${i}`,
+      assets: {
+        create: {
+          label: `listingEvents: ${i} label: ${i}`,
+          fileId: `listingEvents: ${i} fileId: ${i}`,
+        },
+      },
+    },
+  },
+  listingImages: {
+    create: {
+      ordinal: 1,
+      assets: {
+        create: {
+          label: `listingImages: ${i} label: ${i}`,
+          fileId: `listingImages: ${i} fileId: ${i}`,
+        },
+      },
+    },
+  },
+  listingMultiselectQuestions: {
+    create: [
+      {
+        ordinal: 1,
+        multiselectQuestions: {
+          create: {
+            text: `multiselectQuestions: ${i} text: ${i}`,
+            subText: `multiselectQuestions: ${i} subText: ${i}`,
+            description: `multiselectQuestions: ${i} description: ${i}`,
+            links: {},
+            options: {},
+            optOutText: `multiselectQuestions: ${i} optOutText: ${i}`,
+            hideFromListing: true,
+            applicationSection:
+              MultiselectQuestionsApplicationSectionEnum.programs,
+            jurisdictions: {
+              connect: {
+                id: jurisdictionId,
+              },
+            },
+          },
+        },
+      },
+      {
+        ordinal: 2,
+        multiselectQuestions: {
+          create: {
+            text: `multiselectQuestions: ${i} text: ${i}`,
+            subText: `multiselectQuestions: ${i} subText: ${i}`,
+            description: `multiselectQuestions: ${i} description: ${i}`,
+            links: {},
+            options: {},
+            optOutText: `multiselectQuestions: ${i} optOutText: ${i}`,
+            hideFromListing: true,
+            applicationSection:
+              MultiselectQuestionsApplicationSectionEnum.preferences,
+            jurisdictions: {
+              connect: {
+                id: jurisdictionId,
+              },
+            },
+          },
+        },
+      },
+    ],
+  },
+  units: unitFactory(i, i, jurisdictionId),
+});
+
+const unitFactory = (
+  numberToMake: number,
+  i: number,
```

Comment:
Same feedback for realistic data

Reply:
I think Imma write up a ticketo to capture our need for better seed data along with the fields we need filled out

I think it would be kinda funny if everyone on the team just made 1 listing and we could use that as seed data. thoughts?
tagging @ludtkemorgan for his thoughts too